Source/WebCore/Modules/reporting/ReportingObserver.cpp

```diff
@@ -41,15 +41,17 @@ namespace WebCore {
 
 WTF_MAKE_ISO_ALLOCATED_IMPL(ReportingObserver);
 
-static bool isVisibleToReportingObservers(const AtomString& type)
+static bool isVisibleToReportingObservers(const String& type)
 {
     static NeverDestroyed<Vector<AtomString>> visibleTypes(std::initializer_list<AtomString> {
         AtomString { "csp-violation"_s },
```

Comment:
Similar issue here I believe?

Reply:
I *think* this is now correct in the current PR. Can you clarify?